syntax = "proto3";

message PhoneBook {
    map<string, string> phones = 1;
}
